packages/cozy-client/src/manifest.js

```diff
@@ -0,0 +1,85 @@
+const AUTHORIZED_CATEGORIES = [
+  'banking',
+  'cozy',
+  'energy',
+  'health',
+  'host_provider',
+  'insurance',
+  'isp',
+  'mes_infos',
+  'online_services',
+  'partners',
+  'press',
+  'productivity',
+  'ptnb',
+  'public_service',
+  'shopping',
+  'social',
+  'telecom',
+  'transport'
+]
+
+/** Filters unauthorized categories. Defaults to ['others'] if no suitable category. */
+export function sanitizeCategories(categories) {
+  if (!categories) return ['others']
+  const filteredList = categories.filter(c => AUTHORIZED_CATEGORIES.includes(c))
+  if (!filteredList.length) return ['others']
+  return filteredList
+}
+
+export function areTermsValid(terms) {
+  return !!terms && !!terms.id && !!terms.url && !!terms.version
+}
+
+export function isParnershipValid(partnership) {
+  return !!partnership && !!partnership.description
```

Comment:
Nit: I think `Boolean(partnership && partnership.description)` is more readable if you absolutely want to return a real boolean.